client/components/AssignmentItem/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ListItem, Checkbox, UserAvatar, AbsoluteDate } from '../index'
+import classNames from 'classnames'
+import { get } from 'lodash'
+import moment from 'moment'
+
+const getCoverageIcon = (type) => {
+    const coverageIcons = {
```

Comment:
As we have this function in PlanningItem/index.jsx, we can probably move this to utils/index.js and use it from there.